### PR TITLE
Add note about sharing diffs to appdev

### PR DESCRIPTION
**Pull request by zachmargolis**
adds new infobox

Screenshot 2024-08-30 at 12 30 34 PM

### CODE REVIEW

**Review by colter-nattrass — APPROVED** [review]
Thanks for the clarification here Zach

**Comment by Cinderella252** [comment]
> adds new infobox
> 
> Screenshot 2024-08-30 at 12 30 34 PM
> add diff to app<dev

File: _articles/devops-scripts.md [hunk 1]
```diff
@@ -55,6 +55,14 @@ app-s3-secret: Upload changes to S3? (y/n)
 y
 ```
 
+{%- capture production_warning -%}
+**Note**: When editing secrets, especially in **production**,
+it's a best practice to share the diff in the `#login-appdev` channel for visibility.
+Make sure to redact sensitive values!
+{%- endcapture %}
```
Comment (Cinderella252) [inline _articles/devops-scripts.md:62]:
```suggestion
- {%- endcapture %}
```

**Review by Cinderella252 — APPROVED** [review]
Diff in app<dev

**Pull request merged**